### PR TITLE
Fix some printing issues

### PR DESCRIPTION
**Pull request by isuruf**
1. Fixed an issue with RealDouble
2. `sqrt(2)` is printed as `sqrt(2)` instead of `2**(1/2)`

### CODE REVIEW

**Comment by certik** [comment]
+1, just fix the formatting issues.

File: symengine/printer.cpp [hunk 1]
```diff
@@ -80,7 +80,7 @@ void StrPrinter::bvisit(const RealDouble &x)
     s.precision(std::numeric_limits<double>::digits10);
     s << x.i;
     str_ = s.str();
-    if (str_.find(".") == std::string::npos) {
+    if (str_.find(".") == std::string::npos and str_.find("e") == std::string::npos) {
```
Comment (abhinavagarwal07) [inline symengine/printer.cpp]:
```c++
   if (str_.find(".") == std::string::npos
        and str_.find("e") == std::string::npos) {
```

File: symengine/printer.cpp [hunk 2]
```diff
@@ -219,6 +219,17 @@ void StrPrinter::bvisit(const Add &x)
     str_ = o.str();
 }
 
+void StrPrinter::_print_pow(std::ostringstream &o, const RCP<const Basic> &a, const RCP<const Basic> &b)
```
Comment (abhinavagarwal07) [inline symengine/printer.cpp]:
```c++
void StrPrinter::_print_pow(std::ostringstream &o, const RCP<const Basic> &a,
                            const RCP<const Basic> &b)
```

**Comment by certik** [comment]
+1 if tests pass.

**Comment by certik** [comment]
Note sure what is going on with the Windows build.

**Pull request merged**